src/commonUtils.js

```diff
@@ -0,0 +1,14 @@
+export function isLetter(s) {
+  if (s.length !== 1) return false;
+  // console.log("checking letter: " + s);
+
+  const charCode = s.charCodeAt();
+  // console.log("charcode: '" + "Z".charCodeAt() + "'");
+  if (
+    (charCode >= "a".charCodeAt() && charCode <= "z".charCodeAt()) ||
```

Comment:
"a".charCodeAt(), is a constant value. Replace it with the constant. Ditto for others.